e2e-tests/src/test/kick_out.rs

```diff
@@ -0,0 +1,158 @@
+use aleph_client::{
+    change_validators, get_committee_kick_out_config, get_current_era_validators,
+    get_current_session, get_kick_out_reason_for_validator, get_next_era_non_reserved_validators,
+    get_next_era_reserved_validators, get_underperformed_validator_session_count,
+    wait_for_at_least_session, wait_for_event, wait_for_full_era_completion, AccountId,
+    SignedConnection, XtStatus,
+};
+use codec::Decode;
+use log::info;
+use primitives::{
+    CommitteeSeats, KickOutReason, SessionCount, DEFAULT_CLEAN_SESSION_COUNTER_DELAY,
+    DEFAULT_KICK_OUT_MINIMAL_EXPECTED_PERFORMANCE, DEFAULT_KICK_OUT_SESSION_COUNT_THRESHOLD,
+};
+
+use crate::{
+    accounts::{account_ids_from_keys, get_validator_seed, NodeKeys},
+    rewards::set_invalid_keys_for_validator,
+    validators::get_test_validators,
+    Config,
+};
+
+const VALIDATOR_TO_DISABLE_NON_RESERVED_INDEX: u32 = 0;
+const VALIDATOR_TO_DISABLE_OVERALL_INDEX: u32 = 2;
+const NODE_TO_DISABLE_ADDRESS: &str = "127.0.0.1:9945";
```

Comment:
I think this line deserves a comment, we had a fun time debugging that it is actually coupled with the infra setup of e2 tests, ie validator //2 has port 9945 etc

Reply:
Added.